server/calendar/scraper.js

```diff
@@ -1,47 +1,58 @@
 'use strict';
 
 const {
-  readDate, readTime,
-  readInfo, readPhotoUrl, readLink
+  readDate, readTime, readInfo,
+  readPhotoUrl, readLink, sanitizeHTML
 } = require('./helpers.js');
 
-module.exports = function ($, $el) {
+const HR_LOCALES = {
+  location: 'Lokacija',
+  date: 'Datum',
+  ticket: 'Ulaznica',
+  category: 'Kategorija'
+}
+
+module.exports = ($, $el) => {
   return $el.find('li a')
     .map((_, el) => readDay($, $(el)))
     .get();
 };
 
 function readDay($, $el) {
-  let href = $el.attr('href');
-  let date = readDate($el.find('.dan'));
+  const href = $el.attr('href');
+  const date = readDate($el.find('.dan'));
 
-  let events = $(href).find('a')
+  const events = $(href).find('a')
     .map((_, el) => readEvent($, $(el)))
     .get();
 
   return { date, events };
 }
 
 function readEvent($, $el) {
-  let href = $el.attr('href');
-  let time = readTime($el.find('.dan'));
+  const href = $el.attr('href');
+  const time = readTime($el.find('.dan'));
 
-  let $event = $(href);
-  let $content = $(href).find('p');
-
-  let title = $event.find('h2').text();
-  let info = readInfo($content.eq(0));
-  let description = $content.eq(1).text();
+  const $event = $(href);
+  const $content = $(href).find('p');
 
-  let event = { title, time, info, description };
+  const title = $event.find('h2').text();
+  const info = readInfo($content.eq(0));
+  const image = readPhotoUrl($event.find('.foto'));
+  const link = readLink($content.eq(1).next('a'));
 
-  // Extract event photo.
-  let image = readPhotoUrl($event.find('.foto'));
-  if (image) event.image = image;
-
-  // Parse read more link.
-  let link = readLink($content.eq(1).next('a'));
-  if (link) event.link = link;
-
-  return event;
+  let description = $content.eq(1).text();
+  description = sanitizeHTML(description);
+
+  return {
+    category: info[HR_LOCALES.category],
+    title,
+    date: info[HR_LOCALES.date],
+    time,
+    location: info[HR_LOCALES.location],
+    ticket: info[HR_LOCALES.ticket],
+    description,
+    image,
+    link
+  };
```

Comment:
Well, you ended up with table so populate it fully by inlining those 5 remaining vars? :bowtie: 
Or move those 4 up?